Makefile.darwin

```diff
@@ -16,7 +16,11 @@
 #darwin specific settings
 COMMANDS += containerd-shim
 
-# amd64 supports go test -race
+# amd64 supports go test -race,
+# but causes race-limit of simultaneously goroutines, thus disabled
```

Comment:
I'd like to keep `-race` here for catching concurrency issues. What is the issue you'd like to workaround?

Reply:
Agree.

I'm sure that this caused the issue of something related to goroutine rate-limit, but now I cannot reproduce this anymore in several cases (use old my PR patchset, downgrade go version, etc).

I'll revert this changes to the original one.

Thanks for the head up.